api/tests/core/users/young_status_test.py

```diff
@@ -19,49 +20,113 @@ def _with_age(age):
 
 class UserStatusTest:
     class EligibleTest:
```

Comment:
(question / style / question de goût) C'était déjà là avant cette PR, mais gagne-t-on vraiment quelque chose à avoir cette classe-dans-une-classe ? Est-ce qu'on va avoir d'autres classes de test dans ce module ? Le module s'appelant `young_status_test.py`, je suppose qu'on n'y mettra pas grand chose de plus que des classes du genre `EligibleTest`. Et donc qu'on pourrait supprimer le niveau `UserStatusTest` et définir la classe `EligibleTest` au niveau du module :

```suggestion
class EligibleTest:
```

On gagnerait un niveau d'indentation (donc de la lisibilité), et on réduirait la longueur des lignes (donc le "risque" d'avoir à passer sur plusieurs lignes du code qui pourrait "tenir" sur une ligne).